src/views/AddApplicationView.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+    import { reactive, ref } from 'vue';
+    import ToolbarComponent from '@/components/ToolbarComponent.vue';
+
+    const tab = ref();
+</script>
+
+<template>
+    <div class="addApplication d-flex flex-column justify-center align-center">
```

Comment:
(cosmetic) "addApplication" doesn't follow the CSS property style. Should be "add-application". Or even better, something more descriptive of its puropse.